engine/chaos/manifests/conf/server-master.toml

```diff
@@ -1,9 +1,13 @@
 etcd-endpoints = ["metastore-business:12479"] # Reuse the same endpoints as the business-metastore.
 
-[frame-metastore-conf]
+[framework-metastore-conf]
```

Comment:
@lichunzhu @CharlesCheung96 Please take care of all configuration changes in this PR, including
- rename `frame-metastore-conf` to `framework-metastore-conf` in configuration file
- Add `schema` configuration(required) in both `framework-metastore-conf` and `business-metastore-conf`